### PR TITLE
removed deprecated jquery's ready function

### LINKED ISSUE #2314: jQuery .ready Deprecated
```
$( document ).ready(function() {
  console.log('*** No no ***');
});
```
Is no longer recommended especially jquery 3+.  Also be aware calling onload bindings inside .ready will cause issues 3+.
Instead use:
```
$(function () {
  console.log('*** Use this instead ***');
 });

window.onload etc.
```

Possible issue for moderate to attack. If someone does take this on remember to not try to consolidate existing event handlers and leave in the same place in scripts workflow ie execution order can be important. 
I've now come across this problem twice when updating jq dependencies to 3.1.1

### PR DESCRIPTION
**Pull request by sachinmukherjee**
This PR Fixes #2314

This PR remove `$( document ).ready(function() {
});` to `$(function () { }); ` as it is no longer recommended in jquery 3+.

@bradymiller Please review this PR

### CODE REVIEW

File: interface/code_systems/dataloads_ajax.php [hunk 1]
```diff
@@ -229,7 +229,7 @@
                 success: function(data) {
                     $(stg_load_id).hide();
                     $(stg_dets_id).html(data);
-                    $(document).ready(function() {});
+                    $(function() {});
```
Comment (bradymiller) [inline interface/code_systems/dataloads_ajax.php]: This seems odd that there is no function in here. I can't think of any reason to do that? Unless I am missing something.

File: interface/forms/fee_sheet/review/initialize_review.js [hunk 2]
```diff
@@ -63,4 +63,4 @@ function initialize_review() {
     get_fee_sheet_options('standard');
     ko.applyBindings(view_model, review.get(0));
 }
-$(document).ready(initialize_review);
+$(initialize_review);
```
Comment (bradymiller) [inline interface/forms/fee_sheet/review/initialize_review.js]: This one doesn't look right. Shouldn't it be?
```
$(function() {
    initialize_review
});
```

Comment (bradymiller) [reply interface/forms/fee_sheet/review/initialize_review.js]: Actually, I could be wrong about this one.

Comment (bradymiller) [reply interface/forms/fee_sheet/review/initialize_review.js]: yep, I'm wrong. Disregard above post :)

**Comment by bradymiller** [comment]
review complete. just noted the odd call that seems to do nothing.

**Comment by sachinmukherjee** [comment]
@bradymiller changed the PR as per your review. Please review this one :-)

**Comment by bradymiller** [comment]
Thanks for the code review @im-Amitto ! 
hi @sachinmukherjee , agree code looks good and bringing into codebase. There are still a couple stragglers left in the codebase; feel free to make another PR with these. Thanks for the improvements! -brady

**Pull request merged**